.github/workflows/actions.yml

```diff
@@ -304,13 +304,182 @@ jobs:
+            create-installer: ${{ startsWith(github.ref, 'refs/tags/') }}
+            installer-suffix: 'win64-installer'
+
+          - job-name: '64-bit MinGW'
+            fftw-arch: 'x64'
+            os-version: '2019'
+            qt-version: '5.15.2'
+            qt-arch: 'win64_mingw81'
+            fftw-url: 'ftp://ftp.fftw.org/pub/fftw/fftw-3.3.5-dll64.zip'
+            cmake-generator: 'MinGW Makefiles'
+            chocolatey-options: '' # '--forcex86' for 32-bit build
+            use-qtwebengine: 'OFF' # might need to be turned off for MinGW
+            ableton-link: 'OFF' # might need to be turned off for MinGW
+            artifact-suffix: 'win64-mingw' # set if needed - will trigger artifact upload
+
+    name: Windows ${{ matrix.job-name }}
+    env:
+      BUILD_PATH: ${{ github.workspace }}/build
+      INSTALL_PATH: ${{ github.workspace }}/build/Install
+      LIBS_DOWNLOAD_PATH: ${{ github.workspace }}/../3rd-party
+      ASIO_URL: 'http://www.steinberg.net/sdk_downloads/asiosdk2.3.zip'
+      ASIO_FOLDER_NAME: ASIOSDK2.3
+      VCVARS_SCRIPT: ${{ matrix.vcvars-script }}
+      VCVARS_SCRIPT_PATH: 'C:/Program Files (x86)/Microsoft Visual Studio/${{ matrix.msvc-year }}/Enterprise/VC/Auxiliary/Build/${{ matrix.vcvars-script }}'
+      ARTIFACT_FILE: 'SuperCollider-${{ needs.lint.outputs.sc-version }}-${{ matrix.artifact-suffix }}'
+      INSTALLER_FILE: 'SuperCollider-${{ needs.lint.outputs.sc-version }}-${{ matrix.installer-suffix }}'
+      CREATE_INSTALLER: ${{ matrix.create-installer }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - name: prepare timestamp for cache
+        id: current-date
+        run: |
+          echo "::set-output name=stamp::$(date '+%Y-%m-%d')"
+          echo "::set-output name=week::$(date '+%V')"
+      - name: cache qt
+        id: cache-qt
+        uses: actions/cache@v1
+        with:
+          path: ../Qt
+          key: ${{ runner.os }}-v1-${{ matrix.os-version }}-${{ matrix.qt-version }}-qt${{ matrix.qt-arch }}
+      - name: install qt using aqtinstall
+        uses: jurplel/install-qt-action@v2
+        with:
+          modules: 'qtwebengine'
+          version: ${{ matrix.qt-version }}
+          arch: ${{ matrix.qt-arch }}
+          cached: ${{ steps.cache-qt.outputs.cache-hit }}
+      - name: install libsndfile
+        run: choco install libsndfile ${{ matrix.chocolatey-options }}
+      - name: download fftw
+        shell: bash
+        env:
+          FFTW_PATH: ${{ env.LIBS_DOWNLOAD_PATH }}/fftw
+        run: |
+          mkdir -p $FFTW_PATH && cd $FFTW_PATH
+          curl -L ${{ matrix.fftw-url }} -o fftw.zip
+          7z x fftw.zip -y
+          # add env var for subsequent steps
+          echo 'FFTW_PATH<<EOF' >> $GITHUB_ENV
+          echo $FFTW_PATH >> $GITHUB_ENV
+          echo 'EOF' >> $GITHUB_ENV
+      - name: create fftw msvc library
+        if: env.VCVARS_SCRIPT
```

Comment:
can this be changed to `if: matrix.vcvars-script`? then you can eliminate `VCVARS_SCRIPT` too.

Reply:
I'll check. There was some weird behavior with using `matrix.` entries in the with `if:` and that's why it was an env var, but I'll try it out.

Reply:
Turns out there was no issue with using `if: matrix.vcvars-script`, thanks for pointing it out. Not sure where I saw issues with this previously.